Functions/Quantification/xASL_quant_ASL.m

```diff
@@ -189,7 +191,7 @@
     if bUseBasilQuantification
         % Here we perform FSL quantification
 		% We pass the path to the image and do all the Image and JSON reading inside the function
-		[PWI, ATT, ABV, Tex] = xASL_quant_FSL(PWI4D_Path, x); 
+		[PWI, ATT, ABV, Tex, ITT] = xASL_quant_FSL(PWI4D_Path, x); 
```

Comment:
* Rik said that we actually remove FSL results and suggested just moving the ITT file before we remove that folder (FSL_last or something). Why do we now load it in and then save it, is it not easier to move the files? Similar to the other derivatives from FSL? Or do we need them anyway in subsequent modeling ourselves?

Reply:
That's a question to you - you creating this function in this way :)

But yes, we need to load them anyway - or at least PWI - to apply scalings and save again. After BASIL, it is still only PWI and M0 scaling needs to be applied. So we should keep it this way. And for simplicity, lets keep Tex, ITT also like that.

Reply:
OK